crates/vector-store/src/httpserver.rs

```diff
@@ -7,17 +7,19 @@ use crate::HttpServerAddr;
 use crate::engine::Engine;
 use crate::httproutes;
 use crate::metrics::Metrics;
+use crate::node_state::NodeState;
 use std::net::SocketAddr;
 use std::sync::Arc;
 use tokio::net::TcpListener;
 use tokio::sync::Notify;
 use tokio::sync::mpsc;
 use tokio::sync::mpsc::Sender;
 
-pub(crate) enum HttpServer {}
+pub enum HttpServer {}
```

Comment:
```suggestion
pub(crate) enum HttpServer {}
```
Do we need a change in visibility?